src/test/resources/com/puppycrawl/tools/checkstyle/checks/upperell/InputUpperEllSemantic.java

```diff
@@ -93,7 +93,7 @@ void exHandlerTest()
     }
 
     /** test **/
-    private static final long IGNORE = 666l + 666L; // violation
+    private static final long IGNORE = 666l + 666L; // violation: Using lowercase 'l' as a suffix for long literals can lead to confusion with the digit '1'. Use 'L' instead.
```

Comment:
```suggestion
    private static final long IGNORE = 666l + 666L; // violation, Should use uppercase ''L''
```
  and you should do the same in the docs [Example](https://github.com/checkstyle/checkstyle/blob/master/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/upperell/Example1.java) file also files of UpperEl